helper/restore_helper.go

```diff
@@ -251,12 +307,12 @@ func doRestoreAgent() error {
 				// Close file before it gets overwritten. Free up these
 				// resources when the reader is not needed anymore.
 				if reader, ok := readers[contentToRestore]; ok {
-					reader.fileHandle.Close()
+					reader.getFileHandle().Close()
```

Comment:
why not
```suggestion
					reader.closeFileHandle()
```

Reply:
> why not
there is no closeFileHandle() in RestoreReader and there was an intention to keep changes smaller

Reply:
> there is no closeFileHandle() in RestoreReader

add new function
```go
func (r *RestoreReader) closeFileHandle() {
	r.fileHandle.Close()
}
```
instead new function `getFileHandle`